src/Foundation/.gitignore

```diff
@@ -1,2 +0,0 @@
-*~
```

Comment:
Looks like someone uses vi but place it in the local dir. I do use vi, but it wasn't me :P